### ISSUE TITLE
Splinter Cell: Double Agent Eye Adaptation Issue (SW Only)

### ISSUE THREAD

**Issue by AmbientMalice** [PCSX2/pcsx2#2006]
***PCSX2 version:***
PCSX2 1.5.0-20170701115459

***PCSX2 options:***
Defaults.

***Plugins used:***
Defaults.

***Description of the issue:***
This is a bit of an odd issue, and a bit subtle. There's a slim chance that it's a bug in the hardware renderer, but I'm inclined to think the SW renderer is at fault because the effect seems too deliberate, and too fitting with Ubisoft's penchant for PS2 graphics trickery.

Splinter Cell: Double Agent seemingly has an eye adaptation effect. Sam's eyes adjust to darkness and light. See the following, taken with the GL backend seconds apart.

![gsdx_20170707210627](https://user-images.githubusercontent.com/10704701/28044586-9fb335ee-661b-11e7-963e-a891944c7a61.png)
![gsdx_20170707210629](https://user-images.githubusercontent.com/10704701/28044587-9fb90938-661b-11e7-9ed1-0ece0780a98d.png)

A good example of the problem is in the first level. When you walk into the cave, and then exit the cave, with the software renderer, the horizon remains seared with bloom. With the hardware renderer, this gradually fades away.

![gsdx_20170711093816](https://user-images.githubusercontent.com/10704701/28044802-d9918e36-661c-11e7-9db8-20270b66e843.png)
![gsdx_20170711093827](https://user-images.githubusercontent.com/10704701/28044803-d9a972c6-661c-11e7-8414-16d4cca2cc13.png)

Basically, Sam's eyes never adjust to changing environments with the software renderer.

***How to reproduce the issue:***
Start singleplayer game, skip cutscenes, enter cave, walk down it a bit, and then turn around and exit cave.

***Last known version to work:***
Unknown.

***PC specifications:***
Intel i5 4670K, Nvidia GTX 750Ti, 16GB RAM, Windows 10.

**Comment by MrCK1** [comment]
A GS dump would probably help in the future...

**Comment by AmbientMalice** [comment]
It'd have to be a pretty large GS dump, wouldn't it?

**Comment by MrCK1** [comment]
A moderately sized one may be fine, but yes probably a little larger.

**Comment by gregory38** [comment]
Honestly it would be hard to fix. It is likely related to rounding/truncating of some operations. Likely SotC menu light effect.

**Comment by prafullpcsx2** [comment]
Fixed with https://github.com/PCSX2/pcsx2/commit/a0bbdcefb32b45d999414c003417c8b5a3ca3e96

**Comment by RedDevilus** [comment]
https://github.com/PCSX2/pcsx2/issues/2006 (to atleast let it be mentioned on the PR)
